libs/components/forms/testing/src/form-error/form-error-harness.ts

```diff
@@ -19,15 +19,10 @@ export class SkyFormErrorHarness extends SkyComponentHarness {
     return SkyFormErrorHarness.getDataSkyIdPredicate(filters);
   }
 
-  async #getFormErrorClasses(): Promise<string[]> {
-    const formErrorClasses = await (await this.host()).getProperty('classList');
-    return Array.from(formErrorClasses);
-  }
-
   /*
```

Comment:
Can you double-check the JSDocs comments in this file? I think the format needs to be the following in order for docs-tools to pick it up (I could be wrong):
```
/**
 *
 */
```